arrows/core/applets/CMakeLists.txt

```diff
@@ -20,12 +20,6 @@ set( headers
 # Add our plugin path to the default list
 include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
 
-###
-# Add path to applets so plugin loader can find them
```

Comment:
Not sure about removing these SKBUILD blocks. Ameya may need to chime in, unless I'm missing that they moved somewhere else already. He put those in to control the wheel build.

Reply:
I think this may be a merge issue. The intent is to consolidate where and how the paths are calculated and communicated to the run-time into one place in the CMake file and in one include file.